kong/router/expressions.lua

```diff
@@ -19,8 +19,28 @@ local PROTOCOLS_OVERRIDE = {
 }
 
 
-local function get_exp_and_priority(route)
+-- net.port => net.dst.port
+local function verify_expression(route)
   local exp = route.expression
+
+  if not exp then
+    return nil
+  end
+
+  if exp:find("net.port", 1, true) then
```

Comment:
This is not robust enough. For example:

```
http.header.host ="net.port.com"
```

(Just to illustrate.)

Reply:
I prefer to do this on the syntax level rather than a string replacing.

Reply:
I think that it should be a corner case.

Reply:
> I think that it should be a corner case.

And we need to cover corner cases.